mathesar/tests/integration/test_toggle_button.py

```diff
@@ -0,0 +1,24 @@
+from playwright.sync_api import expect
+
+from mathesar.tests.integration.utils.locators import get_table_entry, get_tables_list
+
+
+def test_toggle_button(page, base_schema_url):
+    page.goto(base_schema_url)
+    expect(get_tables_list(page)).to_be_empty()
+    page.click("[aria-label='New Table']")
+    page.click("button:has-text('Import Data')")
+    page.click("text=Copy and Paste Text")
+    page.fill("textarea", "foo\ntrue\nfalse\nfalse\ntrue")
```

Comment:
@seancolsen While running this line on playwright inspector playwright treats it as a text and hence the type becomes TEXT while running it locally while I do the same thing locally the browser treats this as a BOOLEAN type. Can you plzzz tell me how to tackle this situation 

Playwright Inspector
![Screenshot from 2022-03-29 14-55-26](https://user-images.githubusercontent.com/56436023/160579954-43b04cc4-7e09-43be-901a-09b3de15a8b5.png)
Browser
![Screenshot from 2022-03-29 14-55-34](https://user-images.githubusercontent.com/56436023/160580027-e03874ed-f453-4d72-be6d-7257f328143b.png)

Reply:
My understanding is that #1168 is supposed to fix that. Are you running with the latest code?

Reply:
@seancolsen Yes I am running it with the latest code.